src/internal/RequestFlowBuilder.ts

```diff
@@ -115,15 +123,71 @@ export class RequestFlowBuilder {
         return res;
     }
 
-    withScript(action: (sb: ScriptBuilder) => void): RequestFlowBuilder {
-        this.buildScript = action;
+    withDefaults(): RequestFlowBuilder {
```

Comment:
i think docs are required for all functions

Reply:
ok, I'll add them